File: Python/platziPy/CursoPOOUber/Python/uberPool.py
from car import Car #Importo la clase padre

class UberPoll(Car): #Creo una nueva clase heradada de la lcase CAR (Car)
    brand   = str
    model   = str

    #Metodo Constructor
    def __init__(self, license, driver, brand, model):
        super().__init__(license, driver)
        self.brand = brand
        self.model = model

File: Python/platziPy/CursoPOOUber/Java/UberPool.java
class UberPool extends Car {
    String brand;
    String model;

    public UberPool(String license, Account driver, String brand, String model) {
        super(license, driver); //Super Hace referencia a los atributos y metodos de la clase padre
        this.brand = brand; //This Hace referenica a los emtodos y atributos de esta clase, hija
        this.model = model;
    }
    
}
